<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-R3U6M9V8" ignored="1" name="round1" tests="11" failures="2" timestamp="31 三月 2018 09:12:35 GMT" time="0.068" errors="0">
  <testcase name="test1" time="0.014" classname="com.pinke.test.demo.TestNGDemo1">
    <failure type="java.lang.AssertionError" message="expected [11] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [11] but found [10]
	at com.pinke.test.demo.TestNGDemo1.test1(TestNGDemo1.java:10)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test6" time="0.001" classname="com.pinke.test.demo.TestNGDemo1">
    <failure type="java.lang.AssertionError" message="expected [11] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [11] but found [10]
	at com.pinke.test.demo.TestNGDemo1.test6(TestNGDemo1.java:38)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test6 -->
  <testcase name="test4" time="0.001" classname="com.pinke.test.demo.TestNGDemo1"/>
  <testcase name="test4" time="0.001" classname="com.pinke.test.demo.TestNGDemo1"/>
  <testcase name="test4" time="0.001" classname="com.pinke.test.demo.TestNGDemo1"/>
  <testcase name="test4" time="0.0" classname="com.pinke.test.demo.TestNGDemo1"/>
  <testcase name="test5" time="0.002" classname="com.pinke.test.demo.TestNGDemo1"/>
  <testcase name="test5" time="0.004" classname="com.pinke.test.demo.TestNGDemo1"/>
  <testcase name="test5" time="0.005" classname="com.pinke.test.demo.TestNGDemo1"/>
  <testcase name="test5" time="0.002" classname="com.pinke.test.demo.TestNGDemo1"/>
  <testcase name="test2" time="0.001" classname="com.pinke.test.demo.TestNGDemo1"/>
  <testcase name="test3" time="0.0" classname="com.pinke.test.demo.TestNGDemo1">
    <ignored/>
  </testcase> <!-- test3 -->
</testsuite> <!-- round1 -->
